src/js/app/app.js

```diff
@@ -0,0 +1,66 @@
+// eslint-disable-next-line import/no-mutable-exports
+import $ from 'js/dom/dom';
+import 'js/modalToggle';
+import youtubeSearch from 'js/youtubeSearch';
+import youtubeArticle from 'js/components/youtubeArticle';
+
+// global state
+let playListState = {
+  // will, watched, like 세가지 상태를 다룰 예정
+  status: 'will',
+  list: JSON.parse(localStorage.getItem('playList'))?.list ?? [],
+};
+
+const setPlayListState = newPlayListState => {
+  playListState = newPlayListState;
+};
+
+$.playListLength.textContent = playListState.list.length;
+
+// 플레이리스트를 업데이트 시켜 줄 render 함수
+const render = newPlayList => {
+  if (newPlayList) setPlayListState({ ...playListState, list: newPlayList });
+  $.mainVideoWrapper.innerHTML = '';
+  if (playListState.list.length) {
+    const $fragment = document.createDocumentFragment();
+    playListState.list.forEach(item => {
+      if (playListState.status === 'will') {
+        if (!item.isWatch) {
+          $fragment.appendChild(youtubeArticle(item, playListState, render));
+        }
+      } else if (playListState.status === 'watched') {
+        if (item.isWatch) {
+          $fragment.appendChild(youtubeArticle(item, playListState, render));
+        }
+      }
```

Comment:
여기는 이렇게 줄일 수 있지 않을까용 ?? ㅎㅎ
```suggestion
      if (playListState.status === 'will' && !item.isWatch) {
           $fragment.appendChild(youtubeArticle(item, playListState, render));
      } else if (playListState.status === 'watched' && item.isWatch) {
        $fragment.appendChild(youtubeArticle(item, playListState, render));
      }
```

Reply:
마지막쯤 급하게 작성한 코드인데 작성하면서도 정말 못생겼더라고요ㅎㅎ 바꾸신대로 보니까 이쁘고 가독성도 너무 좋네요 감사합니다!